packages/sdk/src/controllers/VariableController.ts

```diff
@@ -596,6 +596,29 @@ export class VariableController {
         return this.applyPrefixSuffixDeltaUpdate(id, update);
     };
 
+    /**
+     * Sets the private data for any variable
+     * @param id the id of the variable to update
+     * @param privateData the private data
+     * @returns
+     */
+    setPrivateData = async (id: string, privateData: Record<string, string>) => {
+        const res = await this.#editorAPI;
+        return res
+            .setVariablePrivateData(id, JSON.stringify(privateData))
+            .then((result) => getEditorResponseData<null>(result));
+    };
+
+    /**
+     * Gets the private data for any variable
+     * @param id the id of the variable
+     * @returns the private data
+     */
+    getPrivateData = async (id: string) => {
+        const res = await this.#editorAPI;
+        return res.getVariablePrivateData(id).then((result) => getEditorResponseData<null>(result));
```

Comment:
```suggestion
        return res.getVariablePrivateData(id).then((result) => getEditorResponseData<PrivateData>(result));
```

Reply:
should the setter also not return it?

Reply:
Looking at other setters, it seems we mostly (only?) return when an id is generated from an entity we created. Here the setter is a full replace of the private data so I don't think it has much value.